src/platform/silabs/wifi/wfx_host_events.h

```diff
@@ -94,34 +95,52 @@ typedef struct __attribute__((__packed__)) sl_wfx_mib_req_s
     sl_wfx_mib_req_body_t body;
 } sl_wfx_mib_req_t;
 
-#else /* End WF200 else RS911x */
+#else /* End WF200 else RS9116,917 NCP and 917 SoC */
 
 #include "wfx_msgs.h"
 
 #if (SLI_SI91X_MCU_INTERFACE | EXP_BOARD)
 #include "sl_si91x_types.h"
 #include "sl_status.h"
 #include "sl_wifi_constants.h"
+#include "stdbool.h"
 
 #include "rsi_common_apis.h"
 #include "sl_wifi_device.h"
 
-#define SL_WIFI_ALLOCATE_COMMAND_BUFFER_WAIT_TIME_MS 1000
-#endif
+#define SL_WIFI_ALLOCATE_COMMAND_BUFFER_WAIT_TIME_MS (1000)
+#endif //(SLI_SI91X_MCU_INTERFACE | EXP_BOARD)
 
 /* Wi-Fi events*/
-#define SL_WFX_STARTUP_IND_ID 1
-#define SL_WFX_CONNECT_IND_ID 2
-#define SL_WFX_DISCONNECT_IND_ID 3
-#define SL_WFX_SCAN_COMPLETE_ID 4
-#endif /* WF200 */
+#define SL_WFX_STARTUP_IND_ID (1)
+#define SL_WFX_CONNECT_IND_ID (2)
+#define SL_WFX_DISCONNECT_IND_ID (3)
+#define SL_WFX_SCAN_COMPLETE_ID (4)
+#endif /* WF200_WIFI */
+
+#if (SLI_SI91X_MCU_INTERFACE | EXP_BOARD)
```

Comment:
the block below can be included in the above `#else` part if I am not wrong.

Reply:
Refactored in a cleaner way for 917 SoC and NCP